models/quickbooks__general_ledger.sql

```diff
@@ -96,15 +98,17 @@ with gl_union as (
 ),
 
 accounts as (
+
     select *
     from {{ ref('int_quickbooks__account_classifications') }}
 ),
 
 
 adjusted_gl as (
     select
-        {{ dbt_utils.generate_surrogate_key(['gl_union.transaction_id', 'gl_union.index', 'accounts.name', ' gl_union.transaction_type']) }} as unique_id,
+        {{ dbt_utils.generate_surrogate_key(['gl_union.transaction_id', 'gl_union.index', 'accounts.name', ' gl_union.transaction_type', 'gl_union.transaction_source']) }} as unique_id,
```

Comment:
I think I may have missed where this addition stems from and why it may need to be included here, could you explain this change to me?

Reply:
Unique tests were failing on the `unique_id` when I tried setting this up. Adding the `transaction_source` to the surrogate key fixed this. Should I add to the Changelog?